CRM/Dedupe/Merger.php

```diff
@@ -960,7 +957,7 @@ public static function skipMerge($mainId, $otherId, &$migrationInfo, $mode = 'sa
    *
    * @return bool
    */
-  static protected function addressIsSame($mainAddress, $comparisonAddress) {
+  static public function addressIsSame($mainAddress, $comparisonAddress) {
```

Comment:
@eileenmcnaughton Rename this function since it's not just checking addresses any more. Could work for any entity/api result 'locationEntityIsSame' or something like that. Likewise, rename the params.